src/main/java/com/synopsys/integration/alert/web/api/user/UserActions.java

```diff
@@ -187,11 +199,12 @@ private void validateUserExistsByName(List<AlertFieldStatus> fieldErrors, String
         userModel.ifPresent(user -> fieldErrors.add(AlertFieldStatus.error(FIELD_KEY_USER_MGMT_USERNAME, "A user with that username already exists.")));
     }
 
-    private void validateUserExistsById(List<AlertFieldStatus> fieldErrors, Long userId, String userName) {
-        validateRequiredField(FIELD_KEY_USER_MGMT_USERNAME, fieldErrors, userName);
+    private List<AlertFieldStatus> validateUserExistsById(Long userId, String userName) {
         Optional<UserModel> userModel = userAccessor.getUser(userName);
-        userModel.filter(user -> !user.getId().equals(userId))
-            .ifPresent(user -> fieldErrors.add(AlertFieldStatus.error(FIELD_KEY_USER_MGMT_USERNAME, "A user with that username already exists.")));
+        return userModel.filter(user -> !user.getId().equals(userId))
+                   .map(user -> AlertFieldStatus.error(FIELD_KEY_USER_MGMT_USERNAME, "A user with that username already exists."))
+                   .stream()
+                   .collect(Collectors.toList());
     }
```

Comment:
This looks wrong. If I am validating a user exists, why is there a list of statuses all saying the same thing being returned? Shouldn't it just be a single item returned?